### PR TITLE
WTD Style guide content changes. Related to PR 1501

### PR DESCRIPTION
**Pull request by wabernat**
Edited some content. Fixed several elderly links. Changed some lines to 80-column.

### CODE REVIEW

**Comment by barriebyron** [comment]
@Bradamant3 we worked through all non-accessibility and non-diversity content changes in #1501

File: docs/guide/writing/style-guides.md [hunk 1]
```diff
@@ -1,72 +1,87 @@
...
 
 ## Write your own style guide?
 
-A style guide can be something as simple as a list of decisions you've made about how to refer to different items you frequently write about. Or it can be as complicated as the mighty tomes of major publication houses. 
-
-You can certainly create a style guide of your own. For the sake of simplicity, this approach might work if you're a lone writer or just starting a small docs group. But neither software nor its documentation operates in a vacuum, so it's a good idea to consult other resources as well. Working from an existing style guide can also help you figure out which things matter in your style guide.
+You can certainly create a style guide of your own. This approach might work if
+you're a lone writer or just starting a small docs group. But neither software
+nor its documentation operates in a vacuum, so it's a good idea to consult other
+resources as well. Working from an existing style guide can also help you figure
```
Comment (MarciaRieferJohnston) [inline docs/guide/writing/style-guides.md]:
```suggestion
resources as well. Working from an existing style guide also helps you figure
```

Comment (MarciaRieferJohnston) [reply docs/guide/writing/style-guides.md]: I made this edit just to play with inserting a suggestion. @barriebyron taught me about this feature today at Write the Docs Writing Day. I'll use this all the time at work now! Curious to see exactly how it works.

**Review by plaindocs — APPROVED** [review]
Looks great to me. Thank you! :-)

**Pull request merged**